beacon_node/lighthouse_network/Cargo.toml

```
@@ -44,11 +44,12 @@ prometheus-client = "0.21.0"
 unused_port = { path = "../../common/unused_port" }
 delay_map = "0.3.0"
 void = "1"
+libp2p-quic= { version = "0.9.0-alpha", features=["tokio"]}
```

```suggestion
libp2p-quic= { version = "0.9.1-alpha", features=["tokio"]}
```

Once https://github.com/libp2p/rust-libp2p/pull/4312 is merged and released.

---

Merged and released :heavy_check_mark:

---

thanks Max, updated!